src/styles.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+:root {
+  --font-color: #3D4E61;
+  --border-color: #E1E7ED;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  border: 1px solid var(--border-color);
+  width: 100%;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  color: var(--font-color);
+  box-sizing: border-box;
+  font-family: inherit;
```

Comment:
```suggestion
  font-family: inherit;
  outline: none;
```
<img width="220" alt="Screenshot 2023-12-13 at 12 17 37" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/135232001/a020c28f-f91a-4f1b-8656-4bdd1f1b0c9b">
Consider removing outlines here

Reply:
Corrected.